components/DetailsTabs/DetailsTabs.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div name="details-tabs">
+    <div class="details-tabs__container">
+      <el-row type="flex">
+        <el-col :span="24">
+          <ul class="details-tabs__container--types">
+            <li v-for="tab in tabTypes" :key="tab.label">
+              <nuxt-link
+                class="details-tabs__container--button"
+                :class="{ active: tab.type === 'teamInformation' }"
+                :to="{
+                  name: 'data',
+                  query: {
+                    type: tab.type
+                  }
+                }"
+              >
+                {{ tab.label }}
+              </nuxt-link>
+            </li>
+          </ul>
+        </el-col>
+      </el-row>
+      <div class="details-tabs__container--data">
+        <h3>Project Leader</h3>
+        <p>
+          <u>Kalyanam Shivkumar</u>
+        </p>
+        <h3>Other Contacts</h3>
+        <ul>
+          <li>Charles Fowlkes</li>
+          <li>
+            <u>Viviana Gradinaru</u>
+          </li>
+          <li>Beth Habecker</li>
+          <li>Stephen Daniel</li>
+          <li>
+            <u>David Paterson</u>
+          </li>
+          <li>Irving Zucker</li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'DetailsTabs',
+  data() {
+    return {
+      tabTypes: [
```

Comment:
These will most likely need to be `props` so the parent component can pass down the tab data to this component.

Reply:
@cameronbaney I will address this when I change the component to pass in props in my next PR with contentful.